modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/agg/Accumulators.java

```diff
@@ -873,7 +887,7 @@ private DecimalMinMax(boolean min) {
     }
 
     /** */
-    private static class VarBinaryMinMax implements Accumulator {
+    private static class VarBinaryMinMax<T extends Comparable<T>> implements Accumulator {
```

Comment:
Generalized class is not used. Let's create something like `ComparableMinMax` and create static instances of this class `VARBINARY_MIN_FACTORY`, `VARBINARY_MAX_FACTORY`, `UUID_MIN_FACTORY`, `UUID_MAX_FACTORY`

Reply:
Was not supposed. Removed.